CHANGELOG.md

```diff
@@ -4382,6 +4382,7 @@ Released 2018-09-13
 <!-- begin autogenerated links to lint list -->
 [`absurd_extreme_comparisons`]: https://rust-lang.github.io/rust-clippy/master/index.html#absurd_extreme_comparisons
 [`alloc_instead_of_core`]: https://rust-lang.github.io/rust-clippy/master/index.html#alloc_instead_of_core
+[`allow_attribute`]: https://rust-lang.github.io/rust-clippy/master/index.html#allow_attribute
```

Comment:
Small NIT that I just noticed. Lint names should ideally be plural. Could you rename the lint to `allow_attributes`? We could also choose `allow_attrs` as an abbreviation, but I think the long name is a bit better, since other lints also use the version :)